crates/l2/sequencer/l1_watcher.rs

```diff
@@ -56,214 +54,257 @@ impl L1Watcher {
             l1_block_delay: watcher_config.watcher_block_delay,
         })
     }
+}
 
-    pub async fn run(&mut self, store: &Store, blockchain: &Blockchain) {
-        loop {
-            if let Err(err) = self.main_logic(store, blockchain).await {
-                error!("L1 Watcher Error: {}", err);
-            }
-        }
-    }
+#[derive(Clone)]
+pub enum InMessage {
+    Check,
```

Comment:
What do you think on using "watch" for this?
```suggestion
    Watch,
```